src/Xrefcheck/Verify.hs

```diff
@@ -444,7 +472,7 @@ verifyReference
             let shownFilepath = dropWhile isPathSeparator (toString rLink)
             canonicalPath <- riRoot </ shownFilepath
             checkRef rAnchor riRoot canonicalPath shownFilepath
-          RIExternal -> checkExternalResource config rLink
+          RIExternal -> checkExternalResource (fromList mempty) config rLink
```

Comment:
How about defining `emptyChain :: RedirectChain`?

Reply:
Now that the underlying type will be just `Data.Seq`, I will derive `Monoid` instead. I did't derive it because of the current reversed list representation.

Reply:
Hm, I don't like this idea, because we only want to append links, not join two link chains, so maybe we don't need the monoid instance.

Reply:
Let's go for `emptyChain` then!